scikit-ci.yml

```diff
@@ -26,6 +26,9 @@ before_install:
                 # SETUP_CMAKE_ARGS
                 if arch in ["x86", "x64"]:
                     os.environ["SETUP_CMAKE_ARGS"] = "-DOPENSSL_ROOT_DIR:PATH=/usr/local/ssl " + os.environ["SETUP_CMAKE_ARGS"]
+                # SETUP_BDIST_WHEEL_ARGS
+                if arch == "aarch64":
+                    os.environ["SETUP_BDIST_WHEEL_ARGS"] = "--plat-name %s" % image_name
```

Comment:
Once https://github.com/dockcross/dockcross/pull/420 is integrated, this should be changed to:

```
os.environ["SETUP_BDIST_WHEEL_ARGS"] = "--plat-name %s" % os.environ["AUDITWHEEL_PLAT"]
```

Reply:
@jcfr, [dockcross/dockcross#420](https://github.com/dockcross/dockcross/pull/420) PR is merged now.

Reply:
Great, thanks for the follow up.

The next issue to address is having openssl libraries and header compiled for aarch64, this will address this error:

```
/work/src/Utilities/cmcurl/lib/md5.c:88:25: fatal error: openssl/md5.h: No such file or directory
 #include <openssl/md5.h>
                         ^                   
```

Since the dockcross image host is `x86_64` and the toolchain `aarch64`, I do not think we can install `aarch64` dependencies  using the host the yum package manager, we have to compile them

Reply:
I also wonder if we can download relevant openssl packages from http://mirror.centos.org/centos/8/BaseOS/aarch64/os/Packages/ and manually extract them ?

Reply:
For normal cross-compilation of openssl, I am using below code:
```
export CROSS_COMPILE=aarch64-linux-gnu- && \
export PATH=$PATH:/tmp/toolchain/gcc-arm-8.3-2019.03-x86_64-aarch64-linux-gnu/bin && \
cd /tmp/ && \
git clone https://github.com/openssl/openssl && \
cd openssl && \
./Configure linux-aarch64 --cross-compile-prefix=${CROSS_COMPILE} --prefix=/tmp/openssl/build shared && \
make depend && \
make -j5 && \
make install && \
cp -r /tmp/openssl/build/include/* /tmp/toolchain/gcc-arm-8.3-2019.03-x86_64-aarch64-linux-gnu/aarch64-linux-gnu/libc/usr/include/ && \
cp -r /tmp/openssl/build/lib/* /tmp/toolchain/gcc-arm-8.3-2019.03-x86_64-aarch64-linux-gnu/lib/gcc/aarch64-linux-gnu/8.3.0/
```

In dockcross, I think we need to update **CROSS_COMPILE=aarch64-unknown-linux-gnueabi-** and copy libraries to /usr/xcc/aarch64-unknown-linux-gnueabi/aarch64-unknown-linux-gnueabi/lib folder.

Maybe we can copy from http://mirror.centos.org/centos/8/BaseOS/aarch64/os/Packages , but need to check on what all packages are required.

Reply:
Thanks for the follow up :pray: 

> In dockcross, I think we need to update CROSS_COMPILE=aarch64-unknown-linux-gnueabi- 

Could you check that OpenSSL can be compiled in dockcross ? Once we have the receipt, it will be easy to finalize this PR.

Reply:
@jcfr ,
I am able to successfully build openssl in dockcross/manylinux2014-aarch64 docker container by following below steps:
```
yum install perl-IPC-Cmd.noarch perl-IPC-Run.noarch perl-IPC-Run3.noarch
git clone https://github.com/openssl/openssl && \
cd openssl
./Configure linux-aarch64 --cross-compile-prefix= --prefix=/tmp/openssl/build shared
make depend
make -j5
make install
cp -r /pruthvi/openssl/build/lib/* /usr/xcc/aarch64-unknown-linux-gnueabi/aarch64-unknown-linux-gnueabi/lib64/
cp -r /pruthvi/openssl/build/include/* /usr/xcc/aarch64-unknown-linux-gnueabi/aarch64-unknown-linux-gnueabi/include/
```

Please let me know if any more info is required.

Reply:
Thanks for the update, looking at it now